src/main/java/uk/gov/hmcts/cmc/claimstore/documents/output/PDF.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.cmc.claimstore.documents.output;
+
+public class PDF {
+    public static final String CONTENT_TYPE = "application/pdf";
```

Comment:
`MediaType.APPLICATION_PDF_VALUE` ?

Reply:
That would make the class depend on some external HTTP packages which I did not like. I also think it is readable enough.

Reply:
but the purpose of this is for http? Otherwise why else do you care about the content type?

Reply:
I still think that bringing dependency on HTTP libraries for such a small cause is unnecessary  but I am keen to comply since this is internal API and we are in spring world anyway.

Reply:
Done 👍

Reply:
Done 👍